workers/deployment_manager/ServiceAccountGenerator/ServiceAccountGenerator.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Improbable.SpatialOS.Platform" Version="13.5.1" />
```

Comment:
`"13.5.1"` but the rest code specified `"13.5.0"`

Reply:
doesn't really matter as Worker SDK & Platform SDK are completely separated. but can make them the same version